packages/proto/src/codegen/seiprotocol/client.ts

```diff
@@ -1,53 +1,68 @@
-import { OfflineSigner, GeneratedType, Registry } from "@cosmjs/proto-signing";
```

Comment:
synced with carson offline - we're going to omit the rpc/rest client generation in the codegen and just use `getSigningClient` from the core package